src/main/kotlin/icfp2019/app.kt

```diff
@@ -3,23 +3,24 @@ package icfp2019
 import java.io.File
 
 fun main(args: Array<String>) {
-    File(args[0]).walk().forEach {
+    File("/Users/jswenson/sources/personal/icfp-2019/problems/prob-001.desc").walk().forEach {
```

Comment:
Just pass "/Users/jswenson/sources/personal/icfp-2019/problems/prob-001.desc" as the first arg in your run config in intellij or create a custom task to do this smaller thing in the gradle config.  

Probably should not change this here.
```suggestion
    File(args[0]).walk().forEach {
```